build_tools/kokoro/gcp_ubuntu/simulate_kokoro.sh

```diff
@@ -15,14 +15,15 @@
 # limitations under the License.
 
 # Simulates the behavior of Kokoro on a local machine.
-# Usage:
+# Usage (under sudo):
 #   ./build_tools/kokoro/gcp_ubuntu/simulate_kokoro.sh \
 #     build_tools/kokoro/gcp_ubuntu/bazel/linux/x86-swiftshader/core/build_kokoro.sh
 #
 # Just does the part of the Kokoro setup that we care about and invokes the
 # given build script.
 # An optional second parameter can be used to specify a different repo to clone
-# from. Especially useful for cloning the current git repo.
+# from. Especially useful for cloning the current git repo. If there's any local
+# change, be sure to commit it before running this simulation.
```

Comment:
Add these while testing the Kokoro scripts locally. Please let me know if they should be moved to a markdown doc.

Reply:
Ah good call. Thanks for adding. Comment here is fine :-)